modules/ob-configuring-pods-in-build-run.adoc

```diff
@@ -0,0 +1,21 @@
+// This module is included in the following assembly:
+//
+// * configuring/configuring-build-runs.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ob-configuring-pods-in-build-run_{context}"]
+= Pod configuration
+
+You can use the following optional `BuildRun` CR fields to configure {builds-shortname} pods:
+
+* The `spec.tolerations` field specifies the tolerations for the {builds-shortname} pod with the following limitations:
++
+** Only the `NoSchedule` taint `effect` is supported
```

Comment:
```suggestion
** Only the `NoSchedule` taint `effect` is supported.
```
If possible, add whatever the relevant noun is after `effect`. See "Object references" in the OCP docs guidelines for more information: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references